open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-introduction.ipynb

```diff
@@ -100,21 +100,47 @@
   },
```

Comment:
<p>Such code is not your change. Please exclude them from the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>?</p>